lib/BridgedRoom.js

```diff
@@ -434,6 +446,85 @@ BridgedRoom.prototype.onGitterMessage = function(message) {
     });
 };
 
+BridgedRoom.prototype.handleGitterCommand = async function(fromUser, message) {
+    const IS_USERID = /^@.+:.+$/; // Lazy user id regex.
+    if (message === "!matrix") {
+        message = "!matrix help";
+    }
+
+    // Determine if the user is an admin.
+    await this._listGitterUsers();
+
+    if (!this._gitterAdminList.has(fromUser.username)) {
+        await this._main.getRelayGhost().send(this._gitterRoomName, "Only admin's may use the !matrix command");
+        return;
+    }
+
+
+    const commandParts = message.substr("!matrix".length).split(" ");
+    const command = commandParts[0].toLowerCase();
+    const commandArgs = commandParts.slice(1);
+
+    const roomIds = new Array(this._linkedMatrixRoomIds);
+    if (this._portalMatrixRoomId) {
+        roomIds.push(this._portalMatrixRoomId);
+    }
+
+    let response = "Command not understood.";
```

Comment:
Some of your responses have punctuation and others don't.

Reply:
I think it's consistent now